test/c2chapel/run-c2chapel/PRECOMP

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+
+cfile = str(sys.argv[1]) + '.h'
+filename = str(sys.argv[1]) + '.chpl' # this is the file name with no suffix
+
+c2chapelbin = str(sys.argv[3])[:-4] + 'c2chapel'
+
+# remove file to ensure that it is the output from c2chapel
+os.remove(filename)
+
+# execute c2chapel on the header file and write results to .chpl file
+os.system(c2chapelbin + ' ' + cfile + ' > ' + filename)
+
+# execute c2chapel with gnu extensions as well
+os.system(c2chapelbin + ' --gnu-extensions ' + cfile + ' > ' + filename)
```

Comment:
This is probably fine as-is since it's just a PRECOMP, but if you want to learn some more python:
 - sys.argv should already be strings, so no need to cast
 - almost certainly overkill, but you could look at using `pathlib` to create the filenames instead of manually appending the suffix
 - I don't grok how your `c2chapelbin` building works. I would look at `os.path.dirname` 
 - The `subprocess` module is preferred over `os.system`

Reply:
Alright, changed over to all your suggestions (except the `pathlib` one, where grepping our test dir doesn't yield anything). Also, I added a check to make sure that the output is the same with and without the `--gnu-extensions` flag.